fare/theme/assets/scss/fare/theme.scss

```diff
@@ -10,8 +10,23 @@
 @import "variables";
 @import "../invenio_theme/styles";
 
-.navbar-static-top{
-  background-image: url("../../../static/images/books.png");
+.navbar-default{
+  background-color: transparent;
+  border-color: transparent;
+  background-image: url("../../../static/images/frontpage_library.png");
```

Comment:
This relative path is ugly and doesn't come handy.
We are in SCSS, please try to define the static folder path somewhere at the top and use such a value here if possible.
Something like `options: {includePaths: [path.resolve(__dirname, "static/etcetera/etcetera")]}`